calico/_includes/non-helm-manifests/csi-node-driver.yaml

```diff
@@ -0,0 +1,95 @@
+# This manifest installs the {{site.prodname}} CSI driver which
+# will be installed on every node in the cluster.
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: csi-node-driver
+  namespace: kube-system
+  labels:
+    k8s-app: csi-node-driver
+spec:
+  selector:
+    matchLabels:
+      name: csi-node-driver
+  template:
+    metadata:
+      labels:
+        name: csi-node-driver
+    spec:
+      tolerations:
+      # this toleration is to have the daemonset runnable on master nodes
```

Comment:
I forgot about the change. The version of kubeadm that I was using to test it still references `node-role.kubernetes.io/master` but I'll add both tolerations so that it will work with both for now. We can remove this reference once everything has transitioned.